html/elements/h1.json

```
@@ -39,6 +39,45 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "no_ua_styles_for_h1_in_article_aside_nav_section": {
+          "__compat": {
+            "description": "No user agent styles for `<h1>` in `<article>`, `<aside>`, `<nav>`, and `<section>`.",
```

Given that we're in the context of `<h1>`, we can remove that duplication:

```suggestion
        "no_ua_styles_in_article_aside_nav_section": {
          "__compat": {
            "description": "No user agent styles in `<article>`, `<aside>`, `<nav>`, and `<section>`.",
```